src/components/Edit/index.jsx

```diff
@@ -544,7 +544,8 @@ const Edit = ({
       const selectedDate = dayjs(event.target.value).format(ShortDate);
       let value = calculateDaysFromDate(event.target.value) + 1;
       setInputValue(selectedDate);
-      setEmitValue(value);
```

Comment:
See https://github.com/Dyalog/ewc-client/pull/330/files . When we do a WG for Value we want the days, and for Text we want the literal text.